.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+env:
+  GLUALINT_VERSION: 1.16.2
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
```

Comment:
is there any reason to not call this job `lint` or something similar to better reflect what the job does?